test/Exercism.Analyzers.CSharp.IntegrationTests/ExerciseAnalyzerTests.cs

```diff
@@ -5,13 +5,16 @@ namespace Exercism.Analyzers.CSharp.IntegrationTests
     public class ExerciseAnalyzerTests
     {
         [Theory]
-        [TestSolutionsDataAttribute]
+        [TestSolutionsData]
         public void SolutionShouldBeCorrectlyAnalyzed(TestSolution testSolution)
         {
             var analysisRun = TestSolutionAnalyzer.Run(testSolution);
 
-            Assert.Equal(analysisRun.Expected.Status, analysisRun.Actual.Status);
-            Assert.Equal(analysisRun.Expected.Comments, analysisRun.Actual.Comments);
+            Assert.True(analysisRun.Expected.Status == analysisRun.Actual.Status, testSolution.Directory);
+            foreach(var expectedComment in analysisRun.Expected.Comments)
```

Comment:
I would like to keep the comments matching as it was, as we really want to be sure that the comments that are sent match those that we expect.

This does mean that you'll have to modify quite a bit of the expected analysis output files, but you could use the bulk updater to help you with this by having it analyze the test files. It will then write the output to those files, which you can then use to update the expected analysis output files.